Formula/docker.rb

```diff
@@ -3,7 +3,8 @@ class Docker < Formula
   homepage "https://www.docker.com/"
   url "https://github.com/docker/docker-ce.git",
       :tag      => "v19.03.1",
-      :revision => "74b1e89e8ac68948be88fe0aa1e2767ae28659fe"
+      :revision => "74b1e89e8ac68948be88fe0aa1e2767ae28659fe",
```

Comment:
This comma breaks the build. It shouldn't be there

Reply:
Whoops, bad rebase! Fixed.